test/parallel/test-internal-errors.js

```diff
@@ -114,3 +114,20 @@ assert.throws(() => {
     throw new errors.TypeError('TEST_ERROR_1', 'a');
   }, common.expectsError('TEST_ERROR_1', TypeError, /^Error for testing 2/));
 }, /^AssertionError: error.message does not match/);
+
+
+// // Test ERR_INVALID_ARG_TYPE
+assert.strictEqual(errors.message('ERR_INVALID_ARG_TYPE', ['a', 'b']),
+                   'The "a" argument must be type b');
+assert.strictEqual(errors.message('ERR_INVALID_ARG_TYPE', ['a', ['b']]),
+                   'The "a" argument must be type b');
+assert.strictEqual(errors.message('ERR_INVALID_ARG_TYPE', ['a', ['b', 'c']]),
+                   'The "a" argument must be one of type b, or c');
+assert.strictEqual(errors.message('ERR_INVALID_ARG_TYPE',
+                   ['a', ['b', 'c', 'd']]),
+                   'The "a" argument must be one of type b, c, or d');
+assert.strictEqual(errors.message('ERR_INVALID_ARG_TYPE', ['a', 'b', 'c']),
+                   'The "a" argument must be type b. Received type string');
+assert.strictEqual(errors.message('ERR_INVALID_ARG_TYPE',
+                   ['a', 'b', undefined]),
```

Comment:
Hmm, looks like the linter is OK with arguments not indented the same way, as long as it's indented? Anyway it doesn't need to be fixed, just a nit :D